parts/typeform-popup.php

```diff
@@ -0,0 +1,13 @@
+<?php
+    $style = ($style) ? $style: 'link';
```

Comment:
Could you add a space before the `:`? It kinda looks like a `key: value` at first glance

Reply:
That's true! I will change it